packages/jest-console/package.json

```diff
@@ -25,8 +25,7 @@
     "slash": "^3.0.0"
   },
   "devDependencies": {
-    "@jest/test-utils": "^28.0.0-alpha.8",
-    "@types/node": "*"
+    "@jest/test-utils": "^28.0.0-alpha.8"
```

Comment:
should we use worspace: to reference package in jest monorepo?

Reply:
Yes, however lerna doesn't understand it: https://github.com/lerna/lerna/issues/2564

Reply:
`yarnpkg/berry` and `babel` both use `workspace:` schema to specify internal deps via yarn without lerna, maybe we could achieve that without lerna?

now we only use lerna to publish packages, right?

Reply:
only used to publish, yes